docker-push/action.yml

```diff
@@ -2,23 +2,23 @@ name: docker-push
 description: Push a Docker image to GAR.
 
 inputs:
-  gar_location:
-    description: GCP region where target GAR is located
-    required: true
-    default: us
-  should_authenticate_to_ghcr:
-    description: Whether or not to authenticate to Github Container Registry
-    required: false
-    default: "false"
   image_tags:
-    description: Newline-delimited list of images to be pushed. Typically generated by mozilla-it/deploy-actions/build-image or docker/metadata-action.
+    description: Newline-delimited list of images to be pushed.
     required: true
   workload_identity_pool_project_number:
     description: Project number of workload identity pool used for OIDC authentication.
     required: true
   project_id:
     description: GCP project_id used to construct the service account.
     required: true
+  gar_location:
+    description: GCP region where target GAR is located
+    required: false
+    default: "us"
+  should_authenticate_to_ghcr:
+    description: Whether or not to authenticate to Github Container Registry
+    required: false
+    default: "false"
```

Comment:
I don't understand what this is about. Why would you want to authenticate to GHCR?

Reply:
In the Mozcloud image tagging [proposal](https://docs.google.com/document/d/1mbMD1Fl6aYm69LOL-RA21Fr_hpTIknE1XAqAjxGQrPs/edit?tab=t.0), we chose to support GHCR as a place for teams to:

> publish images [...] as part of an open source project, or if they need a quick way to share an artifact internally without going through the hassle of spinning up a whole new tenant

So, `docker-build` supports tagging an image for GHCR, and `docker-push` supports authenticating to GHCR so the image can be pushed there.

I eventually want to remove this input and automatically authenticate to GHCR if one of the `image_tags` passed to this action includes the `ghcr.io` registry.

Reply:
Got it. I would amend the description to "Whether or not to authenticate to GHCR. Set this to true if one of the image tags is ghcr.io."